deep_utils/dummy_objects/dummy_framework.py

```diff
@@ -1,11 +1,9 @@
 import importlib.util
-import importlib.util
 import os
 import traceback
 from itertools import chain
 from types import ModuleType
 from typing import Any, List, Union
-
```

Comment:
You shouldn't have removed this. this is required because the python libraries and package projects should be separated with a space